Jenkinsfile

```diff
@@ -11,8 +11,6 @@ node('linux && docker') {
       }
 
       stage('Build') {
-        sh '. ./read.version.sh'
-        sh 'echo $PACKAGE_JSON_VERSION'
```

Comment:
These lines were not correctly injecting environment variables, but they are also not needed, so I removed them.